### PR TITLE
Add Person Nationality  on the patient registration page

### PR DESCRIPTION
**Pull request by solemabrothers**
https://app.asana.com/0/1116983304078777/1131177498585346

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/PersonAttributeTypes.java [hunk 1]
```diff
@@ -155,4 +155,34 @@ public String uuid() {
...
+            return 0;
+        }
+
+        @Override
+        public Class<?> format() {
+            return Concept.class;
+        }
+
+        @Override
+        public String name() {
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/aijar/metadata/core/PersonAttributeTypes.java]: @solemabrothers use Title case

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/PersonAttributeTypes.java [hunk 2]
```diff
@@ -155,4 +155,34 @@ public String uuid() {
...
+            return "NATIONALITY";
+        }
+
+        @Override
+        public String description() {
+            return "Nationality of this person";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/aijar/metadata/core/PersonAttributeTypes.java]: @solemabrothers The nationality of the patient

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 3]
```diff
@@ -808,6 +808,12 @@
     <concept  concept_id="165180"  retired="0" datatype_id="2"  class_id="7"  is_set="0"  creator="1"  date_created="2019-07-10 15:49:47"  version="1.0"  changed_by="1"  date_changed="2019-07-10 15:49:47" uuid="2766c090-c057-44f2-98f0-691b6d0336dc"/>
     <concept  concept_id="165179"  retired="0" datatype_id="4"  class_id="11"  is_set="0"  creator="1"  date_created="2019-07-10 15:45:02"  version="1.0"  changed_by="1"  date_changed="2019-07-10 15:45:02" uuid="c09c3d3d-d07d-4d34-84f0-89ea4fd5d6d5"/>
 
+    <!--PERSON ATTRIBUTE CONCEPTS-->
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: @solemabrothers rename this to Nationality Concepts

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 4]
```diff
@@ -3399,6 +3405,12 @@
...
     <concept_name  concept_name_id="160227"  concept_id="165179"  name="Ward"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-10 15:45:02"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="362b39bf-55ff-4974-844d-97bc93dcfdca"/>
 
+    <!--PERSONAL ATTRIBUTE CONCEPTS NAMES-->
+    <concept_name  concept_name_id="160234"  concept_id="165185"  name="Foreigner"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-19 09:11:49"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="f01e3c72-7cdf-449e-9d59-ba5a408f163d"/>
+    <concept_name  concept_name_id="160233"  concept_id="165184"  name="National"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-19 09:10:47"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="6ade7b18-93ca-47a8-ba74-fdf497c93c96"/>
+    <concept_name  concept_name_id="160235"  concept_id="165186"  name="Nationality"  locale="en"  locale_preferred="1"  creator="1"  date_created="2019-07-19 09:13:57"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="95bfc486-e945-4fae-9cd7-fec9c0f44fd1"/>
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: @solemabrothers Did you check if the concepts are already available in CIEL?

Comment (solemabrothers) [reply api/src/main/resources/metadata/Custom_Concepts.xml]: Yes and they were non-existent

**Comment by solemabrothers** [comment]
@ssmusoke i have updated the PR to address all the issues raised

**Comment by slubwama** [comment]
@solemabrothers  can you raise an issue in OpenMRS and Request me to work on in on the issues page

**Pull request closed without merging**